CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 ### Fixed
 
 - `agate/Dropdown` to show focused icon color for Silicon skin
+- `agate/Dropdown` to now show double marquee text
```

Comment:
now? not?Please do careful internal review. 
@stanca-pop-lgp @adrian-cocoara-lgp

Reply:
sorry for that